src/lib.rs

```diff
@@ -462,31 +462,42 @@ impl Database {
         {
             #[cfg(feature = "fontconfig")]
             {
-                self.load_fontconfig();
+                if !self.load_fontconfig() {
+                    log::warn!("Failed to load font dirs via fontconfig.");
+                    log::warn!("Fallback to loading from known font dir paths.");
```

Comment:
Remove this line. One warning line should be enough.

Reply:
Kept the second line.